build.sbt

```diff
@@ -2,7 +2,7 @@ import Dependencies._
 
 val basicSettings = Seq(
   shellPrompt           := { s => Project.extract(s).currentProject.id + " > " },
-  version               := "1.3.2-SNAPSHOT",
+  version               := "1.4.0",
```

Comment:
Let's remove this change.
It doesn't belong in this PR...

Reply:
ok!